pkg/controller/image-stream-tag-reconciler/reconciler.go

```diff
@@ -311,3 +349,17 @@ type nonRetriableError struct {
 func (nre nonRetriableError) Error() string {
 	return nre.err.Error()
 }
+
+const configIndexName = "release-build-config-by-image-stream-tag"
+
+func configIndexFn(in cioperatorapi.ReleaseBuildConfiguration) []string {
+	var result []string
+	for _, istRef := range release.PromotedTags(&in) {
+		result = append(result, fmt.Sprintf("%s/%s:%s", istRef.Namespace, istRef.Name, istRef.Tag))
+	}
+	return result
+}
+
+func configIndexKeyForIST(ist *imagev1.ImageStreamTag) string {
+	return ist.Namespace + "/" + ist.Name
```

Comment:
Why doesn't this use the tag?

Reply:
I don't understand the comment, which tag do you mean?

Reply:
`ist.Name` is the `name:tag`, disregard